### PR TITLE
Fixed issues after creating dist

### PR DESCRIPTION
**Pull request by DominicBoettger**
Variable names were mixed up. Restructured the code.

### CODE REVIEW

File: src/tasks/ngHtml2Js.js [hunk 1]
```diff
@@ -49,37 +49,39 @@ class NgHtml2JsTask {
 	defineTask(gulp) {
 		let options = this.options;
 		gulp.task(options.taskName, options.taskDeps, () => {
-			if(this.options.ngHtml2Js.extension === '.ts') {
-				let chain = gulp.src(options.src)
+			let chain;
+			if(this.options.ngHtml2Js && this.options.ngHtml2Js.extension === '.ts') {
```
Comment (lookfirst) [inline src/tasks/ngHtml2Js.js]: if (

File: src/tasks/ngHtml2Js.js [hunk 2]
```diff
@@ -49,37 +49,39 @@ class NgHtml2JsTask {
...
+				chain = gulp.src(options.src)
+				 .pipe(cache(options.taskName))
+				 .pipe(plumber())
+				 .pipe(changed(options.dest, {extension: '.html'}))
+				 .pipe(htmlMin(options.minimize))
+				 .pipe(ngHtml2Js(options.ngHtml2Js))
+				 .pipe(insert.prepend(options.prepend))
+				 .pipe(to5(options.compilerOptions));
 			}
+			if(chain) {
```
Comment (lookfirst) [inline src/tasks/ngHtml2Js.js]: if (

**Comment by lookfirst** [comment]
Thanks for the contributions! Just small nits on formatting. =)

**Comment by DominicBoettger** [comment]
Thanks for the review and i fixed the formatting :)

**Comment by lookfirst** [comment]
@DominicBoettger Sorry, but there is two if statements that needed fixing. =) You got one but missed the second one. I'm anal about formatting, sorry!

**Comment by DominicBoettger** [comment]
Sorry, was too late yesterday ;-)

**Comment by DominicBoettger** [comment]
Hi @lookfirst, can you please merge the changes?

**Comment by lookfirst** [comment]
@DominicBoettger Sorry about that, I was down in Puerto Vallarta, MX and had terrible internet access. =(

**Comment by DominicBoettger** [comment]
Thanks @lookfirst. I will write some lines about my integration soon.... I think it woul be great if i had the option to go to MX :-) Hope you had a great time. Thanks again for the merge!

**Pull request merged**